test/e2e-seed-data-framework/scripts/setup.js

```diff
@@ -0,0 +1,52 @@
+import { closeConnections } from '../connections.js'
+import { testId, testPrefix } from '../helpers.js'
+import * as ServiceDirectory from '../models/service-directory-models.js'
+
+try {
+  await setup()
+} catch (error) {
+  console.error('Unable to run setup:', error)
+} finally {
+  await closeConnections()
+}
+
+/**
+ * Runs the test data setup scripts.
+ */
+async function setup () {
+    console.log('Executing setup...')
```

Comment:
Over indenting aside - does this framework provide a logging api at all rather than relying on console.log?

Reply:
It doesn't, and I'd argue we don't need to add one. We really only care about making sure logs are sent to the standard output in whatever environment this runs under (console out locally, GitHub Action output in pipelines); I don't see a benefit in adding a custom logger personally